mitodl/open-discussions
# Audio player skip forward / backward progress reset bug fix

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/open-discussions/issues/2845

#### What's this PR do?
Currently, the audio player is using Amplitude's `skipTo` function to skip to a specific point in the audio file.  Apparently, this causes Amplitude to re-load the audio file, causing it to temporarily reset its progress, detailed here:

https://github.com/521dimensions/amplitudejs/issues/384

Instead, I set these buttons to simply calculate the percentage played to set the audio to and use `Amplitude.setSongPlayedPercentage` like the seek bar is using because it doesn't cause the audio to reload.

#### How should this be manually tested?
Play a podcast, press the forward 30 / back 10 buttons and ensure that the progress does not blank out / reset to 0:00.
